I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/treppenhaus2.rpy", line 25, in script
    scene treppenhaus2leiche
  File "renpy/common/000window.rpy", line 95, in _window_auto_callback
    _window_hide()
  File "renpy/common/000window.rpy", line 85, in _window_hide
    renpy.with_statement(trans)
IOError: Couldn't find file 'atropos_will_Kloth_toeten.mov'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/treppenhaus2.rpy", line 25, in script
    scene treppenhaus2leiche
  File "C:\Users\Tatiana Krupenina\Desktop\Renpy\renpy-7.3.5-sdk\renpy\ast.py", line 1269, in execute
    statement_name("scene")
  File "C:\Users\Tatiana Krupenina\Desktop\Renpy\renpy-7.3.5-sdk\renpy\ast.py", line 45, in statement_name
    i(name)
  File "renpy/common/000window.rpy", line 95, in _window_auto_callback
    _window_hide()
  File "renpy/common/000window.rpy", line 85, in _window_hide
    renpy.with_statement(trans)
  File "C:\Users\Tatiana Krupenina\Desktop\Renpy\renpy-7.3.5-sdk\renpy\exports.py", line 1601, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "C:\Users\Tatiana Krupenina\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\core.py", line 2251, in do_with
    clear=clear)
  File "C:\Users\Tatiana Krupenina\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\core.py", line 2702, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "C:\Users\Tatiana Krupenina\Desktop\Renpy\renpy-7.3.5-sdk\renpy\display\core.py", line 3130, in interact_core
    renpy.audio.audio.interact()
  File "C:\Users\Tatiana Krupenina\Desktop\Renpy\renpy-7.3.5-sdk\renpy\audio\audio.py", line 1107, in interact
    periodic()
  File "C:\Users\Tatiana Krupenina\Desktop\Renpy\renpy-7.3.5-sdk\renpy\audio\audio.py", line 1060, in periodic
    six.reraise(exc[0], exc[1], exc[2])
  File "C:\Users\Tatiana Krupenina\Desktop\Renpy\renpy-7.3.5-sdk\renpy\audio\audio.py", line 1038, in periodic_thread_main
    periodic_pass()
  File "C:\Users\Tatiana Krupenina\Desktop\Renpy\renpy-7.3.5-sdk\renpy\audio\audio.py", line 979, in periodic_pass
    c.periodic()
  File "C:\Users\Tatiana Krupenina\Desktop\Renpy\renpy-7.3.5-sdk\renpy\audio\audio.py", line 457, in periodic
    topf = load(self.file_prefix + filename + self.file_suffix)
  File "C:\Users\Tatiana Krupenina\Desktop\Renpy\renpy-7.3.5-sdk\renpy\audio\audio.py", line 73, in load
    rv = renpy.loader.load(fn)
  File "C:\Users\Tatiana Krupenina\Desktop\Renpy\renpy-7.3.5-sdk\renpy\loader.py", line 576, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'atropos_will_Kloth_toeten.mov'.

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Vanitas 1.0
Mon Jun 29 18:11:51 2020
